src/main/java/net/mcreator/element/types/LivingEntity.java

```diff
@@ -139,6 +139,7 @@
 	public Sound hurtSound;
 	public Sound deathSound;
 	public Sound stepSound;
+	public Sound celebrationSound;

```

Comment:
This should have more elaborate name, such as raidCelebrationSound

Reply:
Done!